# Translation of Odoo Server.
# This file contains the translation of the following modules:
# * account_invoice_constraint_chronology
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server 15.0\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-11-08 07:06\n"
"PO-Revision-Date: 2024-11-08 07:06\n"
"Last-Translator: zvERP <galego@zverp.com>\n"
"Language-Team: Xacobe\n"
"Language: gl\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Generator: tradutor Zerever\n"

#. module: account_invoice_constraint_chronology
#: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_journal__check_chronology
msgid "Check Chronology"
msgstr "Comprobar Cronoloxía"

#. module: account_invoice_constraint_chronology
#: code:addons/account_invoice_constraint_chronology/model/account_move.py:0
#, python-format
msgid ""
"Chronology conflict: A conflicting draft invoice dated before {date_invoice}"
" exists, please validate it first."
msgstr ""
"Conflito na cronoloxía: Existe unha factura borrador en conflito con data de"
" factura anterior {dáche_invoice} , por favor valídaa antes de continuar."

#. module: account_invoice_constraint_chronology
#: code:addons/account_invoice_constraint_chronology/model/account_move.py:0
#, python-format
msgid ""
"Chronology conflict: A conflicting validated invoice dated after "
"{date_invoice} exists."
msgstr ""
"Conflito na cronoloxía: Existe unha factura validada en conflito con data de"
" factura {dáche_invoice} posterior."

#. module: account_invoice_constraint_chronology
#: code:addons/account_invoice_constraint_chronology/model/account_move.py:0
#, python-format
msgid ""
"Chronology conflict: An invoice with a higher number {highest_name} dated "
"before {date_invoice} exists."
msgstr ""
"Conflito na cronoloxía: Existe unha factura cun número superior "
"{highest_name} e con data de factura anterior {dáche_invoice}."

#. module: account_invoice_constraint_chronology
#: model:ir.model,name:account_invoice_constraint_chronology.model_account_journal
msgid "Journal"
msgstr "Diario"

#. module: account_invoice_constraint_chronology
#: model:ir.model,name:account_invoice_constraint_chronology.model_account_move
msgid "Journal Entry"
msgstr "Entrada Diaria"

#, python-format
#~ msgid ""
#~ "Chronology Error. Please confirm older draft invoices before {date_invoice} "
#~ "and try again."
#~ msgstr ""
#~ "Erro de cronoloxía. Por favor valida os borradores de facturas anteriores a "
#~ "{dáche_invoice} e tenta de novo."

#, python-format
#~ msgid ""
#~ "Chronology Error. There exist at least one invoice with a later date to "
#~ "{date_invoice}."
#~ msgstr ""
#~ "Erro de cronoloxía. Existe polo menos unha factura con data posterior a "
#~ "{dáche_invoice}."

#~ msgid "Invoice"
#~ msgstr "Factura"
